src/msvc_helper_main-win32.cc

```diff
@@ -113,12 +113,12 @@ int MSVCHelperMain(int argc, char** argv) {
     PushPathIntoEnvironment(env);
   }
 
-  char* command = GetCommandLineA();
-  command = strstr(command, " -- ");
-  if (!command) {
+  std::string utf8_command_line = WideToUtf8(GetCommandLine());
```

Comment:
Why can't this use the argv and argc from main() ?

Reply:
But even if it can't, why does the commandline need to be converted? Seems to me like you could just as easily do this operation using the std::wstring_t returned from GetCommandLineW

Also, make sure you call GetCommandLineW, not GetCommandLine.